chainparams.go

```diff
@@ -89,7 +89,7 @@ var litecoinSimNetParams = litecoinNetParams{
 // regtest on ltc, however litecoind/ltcd has param information for this setup
 var litecoinRegTestNetParams = litecoinNetParams{
 	Params:   &litecoinCfg.RegressionNetParams,
-	rpcPort:  "18334",
+	rpcPort:  "19334",
```

Comment:
This is confusing - what is this used for?

Reply:
LTCD regtest rpcPort value. default port is also 19334. This makes the file consistent as LND uses BTCD or LTCD ports for `chainparams`